src/javascript/app/Modules/Portfolio/Components/empty-portfolio-message.jsx

```diff
@@ -1,12 +1,12 @@
-import React             from 'react';
-import { localize }      from '_common/localize';
-import { IconPositions } from 'Assets/Footer';
+import React        from 'react';
+import { localize } from '_common/localize';
+import Icon         from 'Assets/Footer';
```

Comment:
i thought it would be import from singular index icon file

```suggestion
import Icon         from 'Assets/Common';
```